### DISCUSSION TITLE
Viseron deletes old recordings automatically

### DISCUSSION THREAD

**Discussion by JesusSpiegel** [roflcoopter/viseron#867] [Q&A]
Hi all, 

I am running Viseron on a computer and I found out that I only have the recordings of the last 7/8 days. There are new recordings every day, but somehow Viseron deletes those that are older than 7/8 days old. 

Example: Today is December 29th and the last date I have in the recordings folder is the 22nd. For instance, three days ago a could see the 19th and so forth. It's like Viseron is only keeping the historial of the last 7/8 days. Is that configurable?

Thanks.

**Comment by roflcoopter** [comment]
Hello!

Yes that can be configured by using the `retain` config option under `recorder`

https://viseron.netlify.app/components-explorer/components/ffmpeg/?_highlight=recorder&_highlight=retain
